.github/workflows/run-mas-tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: MAS Unit Tests
+on:
+  push:
+    branches:
+      - main
```

Comment:
I wondered about the same but I simply copied the one of Milo.
.github/workflows/run-tests.yaml

Reply:
Hmm... it's running once we push to main, we don'nt need that at all.
Also, not for the existing workflow. Since we already run the tests on each PR, no need to run it after something is merged